### ISSUE TITLE
Opportunity Role Field is Missing from Contacts

### ISSUE THREAD

**Issue by pstevens71** [salesagility/SuiteCRM#4953]
#### Issue
When you link a contact to an opportunity, there is a field which displays in the contact sub panel "Opportunity Role".  The field is missing from Studio to be able to add it to the contact edit view/detail view so that you can change it.

#### Expected Behavior
The Field should be available in 'Contacts'  in Studio to add the field so you can edit it on the front end.

#### Actual Behavior
The field is actually there, it exists in the database and has a related drop down, but is not visible in Studio.

#### Possible Fix
in:  ../public_html/modules/Contacts/vardefs.php

Change:
'opportunity_role' =>
array(
'name' => 'opportunity_role',
'type' => 'enum',
'source' => 'non-db',
'vname' => 'LBL_OPPORTUNITY_ROLE',
'options' => 'opportunity_relationship_type_dom',
),

to:
'opportunity_role' =>
array(
'name' => 'opportunity_role',
'type' => 'enum',
'source' => 'non-db',
'vname' => 'LBL_OPPORTUNITY_ROLE',
'options' => 'opportunity_relationship_type_dom',
'studio' => true,
),

Adding the last line 'studio' => true,

This now allows the field to be added in studio and is editable on the front end.

#### Steps to Reproduce
1.   Create an Opportunity
2.  Link it to a contact
3.  Try to find the field "opportunity role"
4.  Go to Studio under contacts and try to find the field "opportunity role"
5. Field is nowhere to be found

#### Context
Probably a low priority, not a big deal, but it really takes away form the ability to manage the contacts by role in the opportunity.

#### Your Environment

* SuiteCRM Version used:  7.8.8 LTS and 7.9.7
* Chrome
* MySQL, PHP 7.1
* Apache

**Comment by mattlorimer** [comment]
On the contacts sub panel in opportunities, next to any contact hit 'Edit'.

**Comment by pstevens71** [comment]
Duh!  You know I tried everything to be able to edit except that!  Clicking on the contact, trying to edit the field in line, just couldn't get it.  I guess I figured the edit button did the same as clicking on the contact.

Long story short, it does in fact give you the option to edit the field if you click on the edit button in the opportunity next to the contact.

My solution is not necessary.  Sorry about that.
